de.unistuttgart.iste.sqa.oo.hamstersimulator.parent/pom.xml

```diff
@@ -10,10 +10,15 @@
 
 	<distributionManagement>
 		<repository>
-		  <id>github</id>
-		  <name>GitHub</name>
-		  <url>https://maven.pkg.github.com/SQAHamster/plain-java-hamster</url>  
+			<id>nexus</id>
+			<name>Maven Central</name>
+			<url>http://localhost:8081/repository/maven-releases/</url>  
 		</repository>
+		<snapshotRepository>
+			<id>nexus</id>
+			<name>Maven Central</name>
+			<url>http://localhost:8081/repository/maven-snapshots/</url>  
```

Comment:
Wieso localhost? Das sieht seltsam aus?

Reply:
I forgot to save the file in the editor 🤦 
this was my local test version to a nexus repo